go/vt/servenv/grpc_auth_plugin.go

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Better to rename this to grpc_auth.go. This is because this file non-optional. Any file that's called a 'plugin' is something that can be removed from the build.